src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_x509.c

```diff
@@ -154,7 +154,6 @@ int32_t AndroidCryptoNative_X509ExportPkcs7(jobject* /*X509Certificate[]*/ certs
                                             int32_t* outLen)
 {
     abort_if_invalid_pointer_argument (certs);
-    abort_if_negative_integer_argument (certsLen);
```

Comment:
This seems distinct from disabling a test... should it really be part of the same change?

Reply:
I can throw up a separate PR.  I originally thought it was going to skip android, but realized it was failing for a completely different reason.

Reply:
Ah, OK, so this is fixing Android and disabling it on iOS/tvOS.

Separate changes would be more ideal... but why respin CI?